README.rst

```diff
@@ -30,9 +30,9 @@ To see what coala can do for your language, run:
 
 .. Start ignoring LineLengthBear
 
-======================================= ================================================= ====================================================== =========================================================
-`Official Website <http://coala.io/>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
-======================================= ================================================= ====================================================== =========================================================
+======================================= ===================================================== ================================================= ====================================================== =========================================================
```

Comment:
In the rst source, each link is covered (above and below) by a sequence of `=`, I suspect for aesthetic reasons. This change adds an extra block to accommodate the new link.